starter-code/css/main.css

```
@@ -6,3 +6,109 @@ Green: #00B172
 White: #FFF
 
 */
+
+* {
+    margin: 0;
+    padding: 0;
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+.smallLogo {
+    width:125px;
+}
+
+
+
+header{
+    padding: 1em;
+}
+
+header div{
+    display: flex;
+	flex-direction: row;
+	flex-wrap: nowrap;
+	justify-content: space-between;
+	align-items: stretch;
+	align-content: stretch;
+}
+header ul{
+    display: flex;
+	flex-direction: row;
+	flex-wrap: nowrap;
+	justify-content: space-around;
+	align-items: stretch;
+	align-content: stretch;
+}
+
+header li{
+    list-style: none;
+    padding: 1em;
+}
+
+img{
+    width: 100%;
+    vertical-align: middle;
+}
+
+#section01 {
+    background-image: url("../images/landing.jpg");
+    background-size: cover;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    width:100%;
+    height: 600px;
+    text-align: center;
+    color: #FFF;
+}
+
+#section02{
+    background-color: #00B172;
+    color: #FFF;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+}
+
+#section03{
+    background-color: #00B172;
+    color: #FFF;
+    
+}
+
+#section01 p{
+    font-size: 2em;
+    padding: 0 200px;
+}
+
+#section02 p{
+    font-size: 1.0em;
+    padding: 0 200px;
+}
+
+#section03 p{
+    font-size: 1.0em;
+    padding: 0 200px;
```

Como os atributos do section02 p e do section03 p estão exatamente iguais, uma forma melhor de chamar no CSS é:
```suggestion
#section02 p, #section03 p {
    font-size: 1em;
    padding: 0 200px;
}
```
separando seletores com virgula, significa que você quer atribuir aquelas propriedades para os dois seletores igualmente.